plugins/rikki/heroeslounge/models/Match.php

```diff
@@ -82,6 +83,16 @@ class Match extends Model
         ]
     ];
 
+    public function listWinnerOptions($fieldname, $value, $formData)
```

Comment:
Ideally, this should be part of the controller since it is very specific for the use in backend. Not sure if it can be moved easily. If it stays in the model, please rename it to listParticipatingTeamsForBackend or something like this

Reply:
See below response, as it's somewhat similar.